kubernetes/CHANGELOG.md

```diff
@@ -1,5 +1,10 @@
 # CHANGELOG - kubernetes
 
+1.1.2 / Unreleased
+==================
```

Comment:
Full disclamer I'm not part of the dd team, but that's how I do for the other PRs.